drams/calva_getting_started/hello_repl.clj

```diff
@@ -1,7 +1,7 @@
 (ns hello-repl)
 
-;; == Some VS Code knowledge required ==
-;; This tutorial assumes you know some few things about
+;; == Some VS Code knowledge is required ==
```

Comment:
```suggestion
;; == Some VS Code knowledge required ==
```

This is me playing with words, alluring to expression *Some Assembly Required* 😄